api/dependencies.py

```diff
@@ -47,3 +53,36 @@ def get_datetime_range(datetime_string: str | None) -> Tuple[Timestamp, Timestam
         raise HTTPException(status_code=400, detail=errors)
 
     return start_datetime, end_datetime
+
+
+async def get_current_parameter_names(ttl_hash=None):
+    """
+    This function get a set of standard_names currently in the datastore
+    The ttl_hash should be a value that is updated at the same frequency
+    we want the lru_cache to be valid for.
+    """
+
+    @lru_cache(maxsize=1)
+    async def async_helper(ttl_hash):
+        del ttl_hash  # make linter think we used this value
```

Comment:
If it is only for the linter, I would rather see an inline ignore.

Reply:
Add inline pylint ignore